third_party/openthread/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@
 set(OT_BORDER_AGENT ON CACHE BOOL "enable border agent" FORCE)
 set(OT_BORDER_ROUTER ON CACHE BOOL "enable border router feature" FORCE)
 set(OT_BORDER_ROUTING ${OTBR_BORDER_ROUTING} CACHE BOOL "enable border routing feature" FORCE)
+set(OT_BORDER_ROUTING_NAT64 ${OTBR_TAYGA_NAT64}&&${OTBR_BORDER_ROUTING} CACHE BOOL "enable border routing NAT64 feature" FORCE)
```

Comment:
We may still set OTBR_TAYGA_NAT64=ON when OTBR_TAYGA_NAT64 is OFF, if we are using other NAT64 implementations?

Reply:
I don't quite understand. Do you mean to set `OT_BORDER_ROUTING_NAT64=ON` if we are using any other NAT64 implementation? When we have other implementations later, we can update the condition here to reflect the change.